pkg/applicationserver/io/web/redis/registry.go

```diff
@@ -76,22 +83,25 @@ func (r WebhookRegistry) List(ctx context.Context, ids ttnpb.ApplicationIdentifi
 
 // Set implements WebhookRegistry.
 func (r WebhookRegistry) Set(ctx context.Context, ids ttnpb.ApplicationWebhookIdentifiers, gets []string, f func(*ttnpb.ApplicationWebhook) (*ttnpb.ApplicationWebhook, []string, error)) (*ttnpb.ApplicationWebhook, error) {
-	k := r.Redis.Key(webhookKey, unique.ID(ctx, ids.ApplicationIdentifiers), ids.WebhookID)
+	appUID := unique.ID(ctx, ids.ApplicationIdentifiers)
+	hk := r.Redis.Key(webhookKey, appUID, ids.WebhookID)
```

Comment:
What does `hk` stand for?

Reply:
Hook ID?
I guess `uk` as "unique key" is better fit here